rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java

```diff
@@ -196,6 +197,32 @@ public BuildRecordProvider(
         return restGraphBuilder.from(buildRecordGraph.getGraph(), BuildRecordRest.class);
     }
 
+    public RunningBuildsCountRest getRunningCount() {
+
+        List<BuildTask> x = buildCoordinator.getSubmittedBuildTasks();
+
+        int waitingForDependencies = 0;
+        int running = 0;
+        int enqueued = 0;
+
+        for (BuildTask task: x) {
+            switch(task.getStatus()) {
+                case ENQUEUED:
+                    enqueued += 1;
```

Comment:
Don't be afraid to use enqueued++; ;-)

Reply:
Oh woops you're right! let me know if you want me to change it!

Reply:
done!